src/clj/witan/send/params.clj

```diff
@@ -187,14 +179,41 @@
     (continue-for-latter-ays params academic-years)))
 
 
+(defn- mover-alpha-prior
+  "Calculate prior by adding uniform distribution across allowed settings to observations and normalising."
+  [need-setting valid-transitions valid-settings observations-per-ay ay]
+  (let [[need setting] (s/split-need-setting need-setting)
+        allowed-settings (clojure.set/intersection (set (get valid-settings [ay need]))
+                                                   (set (get valid-transitions setting)))
+        prior (->> (zipmap allowed-settings (repeat (/ 1.0 (count allowed-settings))))
+                   (merge-with + (get observations-per-ay ay))
+                   (#(dissoc % setting)))
```

Comment:
this can be rewritten as:

```clojure
  (-> allowed-settings
      (zipmap (repeat (/ 1.0 (count allowed-settings))))
      ((partial merge-with + (get observations-per-ay ay)))
      (dissoc setting))
```

merge-with is the only thing that annoyingly doesn't play nicely with the `->`